src/main/kotlin/org/rust/lang/core/psi/RsCodeFragment.kt

```diff
@@ -123,6 +120,15 @@ class RsStatementCodeFragment(project: Project, text: CharSequence, context: RsE
     val stmt: RsStmt? get() = PsiTreeUtil.getChildOfType(this, RsStmt::class.java)
 }
 
+class RsReplCodeFragment(fileViewProvider: FileViewProvider, override var context: RsElement)
+    : RsCodeFragment(fileViewProvider, RsCodeFragmentElementType.REPL, context),
+      RsInferenceContextOwner, RsItemsOwner {
+    val stmts: Array<RsStmt> get() = PsiTreeUtil.getChildrenOfType(this, RsStmt::class.java) ?: emptyArray()
```

Comment:
All our code fragments made this way for some reason ;)

Reply:
@vlad20012 Do you mean `PsiTreeUtil.getChildOfType()` usages? These are because there is no extension method `childOfType` yet. I'm going to introduce it and refactor other places (like `org.rust.lang.core.psi.ext.RsMacroKt#getMacroBody`) too after this PR is merged.